callBackPractice/callBackPractice.js

```diff
@@ -59,13 +80,28 @@ contains(foods, 'ribeye', (result) => {
 // Write a function called removeDuplicates that removes all duplicate values from the given array.
 // Pass the array to the callback function.  Do not mutate the original array.
 
+
+const removeDuplicates = (array, cb) => {
+    const changedArray = array.sort();
```

Comment:
Smooth move! I believe, if I'm not mistaken, this helps prevent the original array from being mutated. Nice work!